public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+> 
+> The pace at which the `supporting details` of the party change can be a dial for changing the feel of the game, especially in games where additional mechanical benefits are offered for building up positive `supporting details`.  By default, the intended pace is an average of no more than one detail changing per session.
+
+The `party sheet` holds two clocks which fight against each other, one dragging the party down, and the other pulling the party up. By default, both of these clocks have eight segments. 
+
+### Taking Consequences
+
+The GM or players may suggest that marking a consequence on the `party sheet` makes sense. Generally this should represent a major consequence as it can have long-term implications, and should not be taken lightly - a couple of times per session is probably a good rule of thumb. This adds segments to the danger clock of the party sheet.
+
+> #### Example
+> 
+> The party is in a spaceship being chased by multiple authorities (`supporting detail` “falsified spacecraft registration”). They need to make a quick, clean getaway to get medical equipment to a dying friend. They are in a desperate position - their pilot rolls a 5, indicating success with a consequence. The play group suggests they get away but they’ve pushed their ship to its limits (`supporting detail` “state-of-the-art spacecraft”). The GM agrees - they can take the consequence on their `party sheet` instead of risking the life of their friend. 
+
+### Filling the Danger Clock
+
+When the danger clock fills up the players should decide, based on recent events, what makes the most sense in the storyline:
+Add a new `supporting detail` - the `disposition` of this detail should generally start at `neutral`. Note `supporting details` should all be specific, unique details that tie back into the fiction that are expected to show up again. 
+Downgrade the `disposition` of an existing `supporting detail` (moving from `positive` to `neutral` to ` negative`).
+Remove or replace an existing neutral `supporting detail`. If all the spaces that make sense are taken or negative, consider whether recent events might have completely supplanted an existing `supporting detail`. This could be a good fit for story elements that have served their purpose or didn't really get much play. 
+Create a new immediate threat - the GM exposes a danger and threatens the players with a negative `supporting detail` _unless_ they complete a mission to prevent it.
+
+> #### Example
+> 
+> The danger clock fills from the strain placed on the starship engines racing to help a dying friend.
+> 
+> The party decides it makes the most sense to downgrade the Assets `supporting detail` “state-of-the-art ship” (+) to “damaged ship” (0). 
+
+
+### Marking Progress
+
+Generally players should be given a chance to advance the progress clock a couple of times per session - this can be as the result of completing a project and/or as a reward for successful completion of a mission. 
```

Comment:
suggestion: Marking Progress VS Taking Consequence. The verbiage is different but those are two interwined concepts I believe. I feel like its like "Ticking the PArty Progress Clocks" and "Ticking the Party Danger Clocks"